flow360/component/flow360_params/unit_system.py

```diff
@@ -827,6 +901,41 @@ class Flow360HeatFluxUnit(_Flow360BaseUnit):
     unit_name = "flow360_heat_flux_unit"
 
 
+class Flow360HeatSourceUnit(_Flow360BaseUnit):
+    """:class: Flow360HeatSourceUnit"""
+
+    dimension_type = HeatSourceType
+    unit_name = "flow360_heat_source_unit"
+
+
+class Flow360HeatCapacityUnit(_Flow360BaseUnit):
+    """:class: Flow360HeatCapacityUnit"""
+
+    dimension_type = HeatCapacityType
+    unit_name = "flow360_heat_capacity_unit"
+
+
+class Flow360ThermalConductivityUnit(_Flow360BaseUnit):
+    """:class: Flow360ThermalConductivityUnit"""
+
+    dimension_type = ThermalConductivityType
+    unit_name = "flow360_thermal_conductivity_unit"
+
+
+class Flow360InverseAreaUnit(_Flow360BaseUnit):
```

Comment:
Why do we need extra inverse area and length. Aren't these defined by other quantities?